docs/source/get_involved/governance.rst

```diff
@@ -400,9 +400,11 @@ core developers.
    Ten working days excluding week-ends constitute "reasonable time" in the above sense.
    The period resets at every new change made to the PR.
    It starts only when all GitHub checks pass.
```

Comment:
Just to clarify, the "reasonable time" period is 10 days for non-core devs, and 5 days for core devs (as per the below rule change)?